# set_angle_1.msg

int32[] finger_ids       
int32[] angles      # （0-1000）      


